capstone/pom.xml

```diff
@@ -88,6 +88,12 @@
       <artifactId>mockito-core</artifactId>
       <version>3.4.0</version>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.http-client</groupId>
+      <artifactId>google-http-client-gson</artifactId>
+      <version>1.36.0</version>
+    </dependency>
```

Comment:
Isn't that already in there? Just checking in case it might be duplicated. If it is not then ignore this comment.

Reply:
I think there was a similar dependency for just the HTTP client. That dependency was needed for specifically the gson factory which it turns out I'm not using, haha. I'll remove that dependency.